vowpalwabbit/explore_eval.cc

```diff
@@ -127,13 +127,15 @@ void do_actual_learning(explore_eval& data, multi_learner& base, multi_ex& ec_se
 
   if (label_example != nullptr)  // extract label
   {
-    data.action_label = label_example->l.cb;
-    label_example->l.cb = data.empty_label;
+    data.action_label = std::move(label_example->l.cb);
+    label_example->l.cb = std::move(data.empty_label);
   }
   multiline_learn_or_predict<false>(base, ec_seq, data.offset);
 
   if (label_example != nullptr)  // restore label
-    label_example->l.cb = data.action_label;
+    label_example->l.cb = std::move(data.action_label);
+    data.empty_label.costs.delete_v();
```

Comment:
`delete_v` should no longer be used. Do you mean `clear` instead?

Reply:
Just recommitted with clear